dev-lua/lua-utf8/lua-utf8-0.1.1.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+DESCRIPTION="UTF-8 support to Lua"
+HOMEPAGE="https://github.com/starwing/luautf8"
+SRC_URI="https://github.com/starwing/luautf8/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~mips ~ppc ~ppc64 ~x86"
+IUSE="luajit"
+
+RDEPEND="
+	!luajit? ( >=dev-lang/lua-5.1:= )
+	luajit? ( dev-lang/luajit:2 )"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+DOCS=( README.md )
+
+S="${WORKDIR}/${PN//-/}-${PV}"
+
+src_prepare() {
+	default
+
+	cat > Makefile <<-EOF
+		# this file builds lua-utf8
+
+		LUA_LIBDIR      := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_CMOD $(usex luajit 'luajit' 'lua'))
+		LUA_INC         := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_INC $(usex luajit 'luajit' 'lua'))
+		LUA_SHARE       := ${ED}/$($(tc-getPKG_CONFIG) --variable INSTALL_LMOD $(usex luajit 'luajit' 'lua'))
+		CWARNS          := -Wall -pedantic
+		CFLAGS          += \$(CWARNS) -O3 -I\$(LUA_INC) -fPIC
+		LIB_OPTION      := -shared
+
+		SONAME          := lua-utf8.so
+		SONAMEV         := \$(SONAME).1
+		LIBRARY         := \$(SONAMEV).0.1
+		SRC             := lutf8lib.c
+		OBJ             := \$(patsubst %.c, %.o, \$(SRC))
+
+		FILES           := parseucd.lua
+
+		all: \$(LIBRARY) \$(SONAMEV) \$(SONAME)
+
+		\$(SONAMEV):
+		TABln -s \$(LIBRARY) \$@
+
+		\$(SONAME):
+		TABln -s \$(SONAMEV) \$@
+
+		\$(LIBRARY): \$(OBJ)
+		TAB\$(CC) \$(CFLAGS) \$(LIB_OPTION) -o \$(LIBRARY) \$(OBJ) -lc
+
+		install:
+		TABmkdir -p \$(LUA_LIBDIR)
+		TABcp \$(SONAME) \$(LUA_LIBDIR)
+		TABmkdir -p \$(LUA_SHARE)
+		TABcp \$(FILES) \$(LUA_SHARE)
+
+		clean:
+		TAB\$(RM) \$(LIBRARY) \$(SONAMEV) \$(SONAME) *.o
+	EOF
+
+	sed -i -e 's/TAB/\t/g' Makefile
+}
+
+src_compile() {
+	emake
+}
+
+src_test() {
+	LUA_CPATH=./src/?.so $(usex luajit 'luajit' 'lua') test.lua
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
+
+	einstalldocs
```

Comment:
src_install is redundant here.
this is what default `src_install` does.


```
__eapi6_src_install() {
        if [[ -f Makefile || -f GNUmakefile || -f makefile ]] ; then
                emake DESTDIR="${D}" install
        fi

        einstalldocs
}
```

if you undefine it to use eapi default, leave `DOCS` array global.

Reply:
fixed